### PR TITLE
Fix general bugs

### PR DESCRIPTION
**Pull request by DetroitWhiskey136**

### CODE REVIEW

File: actions/check_if_member_has_role_MOD.js [hunk 1]
```diff
@@ -127,9 +127,9 @@ module.exports = {
     if (role) {
       if (Array.isArray(member)) {
         result = member.every(function every(mem) {
-          return this.dest(mem, 'roles', 'cache') && mem.roles.cache.has(role.id);
+          return mem?.roles?.cache && mem.roles.cache.has(role.id);
```
Comment (almeidx) [inline actions/check_if_member_has_role_MOD.js:130]:
```suggestion
          return mem?.roles?.cache.has(role.id);
```

File: actions/check_if_user_reacted_MOD.js [hunk 2]
```diff
@@ -124,10 +124,8 @@ module.exports = {
 
     let result;
     if (Array.isArray(member)) {
-      result = member.every(
-        (user) => this.dest(reaction, 'users', 'cache') && user && reaction.users.cache.has(user.id),
-      );
-    } else if (this.dest(reaction, 'users', 'cache') && member) {
+      result = member.every((user) => reaction?.users?.cache && user && reaction.users.cache.has(user.id));
```
Comment (almeidx) [inline actions/check_if_user_reacted_MOD.js:127]:
```suggestion
      result = member.every((user) => user && reaction?.users?.cache.has(user.id));
```

File: actions/store_audio_info_MOD.js [hunk 3]
```diff
@@ -201,7 +201,7 @@ module.exports = {
         result = Boolean(Audio.dispatchers[targetServer.id]); // is playing
         break;
       case 2:
-        result = this.dest(Audio.dispatchers[targetServer.id], 'startTime') || 0; // when the music first started playing
+        result = Audio.dispatchers[targetServer.id]?.startTime || 0; // when the music first started playing
```
Comment (almeidx) [inline actions/store_audio_info_MOD.js:204]: All the modifications to this file are irrelevant. It won't work either way

File: actions/store_user_info_MOD.js [hunk 4]
```diff
@@ -170,15 +170,15 @@ module.exports = {
...
+          result = status && status[0]?.name;
         }
         break;
       case 8: // User Custom Status
-        if (this.dest(user.presence, 'activities')) {
-          const status = user.presence.activities.filter((s) => this.dest(s, 'type') === 'CUSTOM_STATUS');
-          result = status && this.dest(status[0], 'state');
+        if (user.presence?.activities) {
+          const status = user.presence.activities.filter((s) => s?.type === 'CUSTOM_STATUS');
+          result = status && status[0].state;
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:181]:
```suggestion
          const status = user.presence.activities.find((s) => s?.type === 'CUSTOM_STATUS');
          result = status?.state;
```

File: actions/store_user_info_MOD.js [hunk 5]
```diff
@@ -170,15 +170,15 @@ module.exports = {
         result = user.lastMessageID;
         break;
       case 7: // User Activities
-        if (this.dest(user.presence, 'activities')) {
+        if (user.presence?.activities) {
           const status = user.presence.activities.filter((s) => s.type !== 'CUSTOM_STATUS');
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:174]:
```suggestion
          const status = user.presence.activities.find((s) => s.type !== 'CUSTOM_STATUS');
```

File: actions/store_user_info_MOD.js [hunk 6]
```diff
@@ -170,15 +170,15 @@ module.exports = {
...
-          result = status && this.dest(status[0], 'name');
+          result = status && status[0]?.name;
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:175]:
```suggestion
          result = status?.name;
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: actions/store_audio_info_MOD.js (hunk 3 above)
Comment (DetroitWhiskey136) [reply actions/store_audio_info_MOD.js:204]: should that file just be removed?

Comment (almeidx) [reply actions/store_audio_info_MOD.js:204]: Yes

File: actions/store_user_info_MOD.js [hunk 7]
```diff
@@ -170,15 +170,15 @@ module.exports = {
...
+          const status = user.presence.activities.find((s) => s.type !== 'CUSTOM_STATUS');
+          result = status?.name;
         }
         break;
       case 8: // User Custom Status
-        if (this.dest(user.presence, 'activities')) {
-          const status = user.presence.activities.filter((s) => this.dest(s, 'type') === 'CUSTOM_STATUS');
-          result = status && this.dest(status[0], 'state');
+        if (user.presence?.activities) {
+          const status = user.presence.activities.find((s) => s?.type === 'CUSTOM_STATUS');
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:180]:
```suggestion
          const status = user.presence.activities.find((s) => s.type === 'CUSTOM_STATUS');
```

File: actions/store_user_info_MOD.js [hunk 8]
```diff
@@ -170,15 +170,15 @@ module.exports = {
         result = user.lastMessageID;
         break;
       case 7: // User Activities
-        if (this.dest(user.presence, 'activities')) {
-          const status = user.presence.activities.filter((s) => s.type !== 'CUSTOM_STATUS');
-          result = status && this.dest(status[0], 'name');
+        if (user.presence?.activities) {
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:173]:
```suggestion
        if (user.presence?.activities.length) {
```

File: actions/store_user_info_MOD.js [hunk 9]
```diff
@@ -170,15 +170,15 @@ module.exports = {
...
+          const status = user.presence.activities.find((s) => s.type !== 'CUSTOM_STATUS');
+          result = status?.name;
         }
         break;
       case 8: // User Custom Status
-        if (this.dest(user.presence, 'activities')) {
-          const status = user.presence.activities.filter((s) => this.dest(s, 'type') === 'CUSTOM_STATUS');
-          result = status && this.dest(status[0], 'state');
+        if (user.presence?.activities) {
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:179]:
```suggestion
        if (user.presence?.activities.length) {
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — APPROVED** [review]

**Pull request merged**